src/components/Map/Map.vue

```diff
@@ -176,7 +176,7 @@ function getOtherMarkersWithSameCoords({
   });
 }
 
-function createOrUpdateMarker({ id, lng, lat, ...properties }: AddMarkerArgs) {
+function addMarker({ id, lng, lat, ...properties }: AddMarkerArgs) {
```

Comment:
What's the thinking on changing this name?

Reply:
I stopped `watch`ing the marker lng/lat, and switched to `onMounted`, so marker updates won't percolate through.